ec2/Vagrantfile

```diff
@@ -109,6 +109,10 @@ if ARGV[0] == "up"
     stop_hours = ENV['STOP_HOURS']
   end
 
+  if ENV['ES_HEAP_SIZE'].nil?
+    ENV['ES_HEAP_SIZE'] = "3"
```

Comment:
we changed this to 3 just so that we could see that it actually changed when we were testing.. now that we know it works, should we change it to 2 instead of 3?

Reply:
Was thinking to leave it to 3gb since we recommend 8gb or larger for the setup. And will avoid changing from 2 to 2 :)

Reply:
cool